tapdb/multiverse.go

```diff
@@ -133,15 +136,27 @@ func (b *MultiverseStore) RootNode(ctx context.Context,
 // RootNodes returns the complete set of known base universe root nodes for the
 // set of base universes tracked in the multiverse.
 func (b *MultiverseStore) RootNodes(ctx context.Context,
-	withAmountsById bool) ([]universe.Root, error) {
+	q universe.RootNodesQuery) ([]universe.Root, error) {
 
 	var (
 		uniRoots []universe.Root
 		readTx   = NewBaseMultiverseReadTx()
 	)
 
+	params := sqlc.UniverseRootsParams{
+		SortDirection: sqlInt16(q.SortDirection),
+		NumOffset:     int32(q.Offset),
+		NumLimit: func() int32 {
+			if q.Limit == 0 {
+				return universe.MaxPageSize
```

Comment:
👍 

We should prob scan elsewhere to implement a default max page size instead of going wumbo by default.

Reply:
@GeorgeTsagk Can you summarize what information you'll need from @Roasbeef to get clarity?

Reply:
> We should prob scan elsewhere to implement a default max page size instead of going wumbo by default.

Are you referring to the iterative page fetching process as scanning? If yes that's done on the client side, and it default to the max page, which is "wumbo" but actually reasonably small (512 items)

Reply:
I mean other places for the pagination calls (existing ones) where we don't have a default max page size. I think non-blocking for this PR tho, can be done in another pass.